minio/policy.py

```diff
@@ -119,6 +134,10 @@ def _new_statements(policy, bucket_name, prefix=''):
             _new_object_statement(policy, bucket_name, prefix))
 
 
+def resource_startswith(prefix, resources):
+    return any([resource.startswith(prefix) for resource in resources])
```

Comment:
is it good idea to use `filter()` function something like 
```py
return filter(lambda r: r.startswith(prefix), resources)
```

I am not sure which one is better.